transport/src/main/java/io/netty/channel/DelegatingChannelPromiseNotifier.java

```diff
@@ -172,6 +172,11 @@ public Void getNow() {
         return delegate.getNow();
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @param mayInterruptIfRunning this value has no effect in this implementation.
```

Comment:
@laosijikaichele actually I think this is not correct as it just delegates it and if it is ignored or not depends on the implementation of `delegate`

Reply:
@normanmaurer  agree, just deleted comment for this method.